articles/node_lifecycle.md

```diff
@@ -27,224 +27,447 @@ Last Modified: {% if page.last_modified %}{{ page.last_modified }}{% else %}{{ p
-While in this state, the node performs any processing, responds to service requests, reads and processes data, produces output, etc.
+#### **Primary State: `Finalized`**
+The `Finalized` state is the state in which the node ends immediately before being destroyed.
+This state exists to support debugging and introspection.
+A node which has failed will remain visible to system introspection and may be potentially introspectable by debugging tools instead of directly destructing.
+There will be no Lifecycle internal/backend policy for automatically restarting a node.
 
-If an error that cannot be handled by the node/system occurs in this state, the node will transition to `ErrorProcessing`.
+*State Assumptions / Behaviors*
+- This state can only be reached via a `ShuttingDown` → `SUCCESS` → `Finalized`
+- Anything the node is deemed responsible for (e.g., Managed Entities) using CPU is/are stopped
+- Anything the node is deemed responsible for (e.g., Managed Entities) using RAM/Mem is/are de-allocated
+- `node->destroy()` can be safely run
 
-#### Valid transitions out of Active
 
-- A node may transition to the `Inactive` state via the `deactivate` transition.
-- A node may transition to the `Finalized` state via the `shutdown` transition.
+**Terminal state**: No Outbound Transitions
 
-### Primary State: Finalized
 
-The `Finalized` state is the state in which the node ends immediately before being destroyed.
-This state is always terminal the only transition from here is to be destroyed.
 
-This state exists to support debugging and introspection.
-A node which has failed will remain visible to system introspection and may be potentially introspectable by debugging tools instead of directly destructing.
-If a node is being launched in a respawn loop or has known reasons for cycling it is expected that the supervisory process will have a policy to automatically destroy and recreate the node.
+#### **Primary State: `UncleanFinalized`**
+This state exists to denote a node that failed recovery in `ErrorProcessing` and therefor the `LifecycleNode` cannot be guaranteed to be cleanly destroyed.
+An external supervisor would therefor be responsible for handling of this node (i.e., keeping it around in an unclean state or cleaning it up such as killing the process it is in).
+
+*State Assumptions / Behaviors*
+- This state can only be reached upon  `ErrorProcessing` → `FAILURE` → `UncleanFinalized` 
+- This state indicates failed recovery and thus is in an unclean / uncertain state w.r.t. Managed Entities / memory / cpu (i.e., node is in an uncertain finalized state where we cannot guarantee it can be cleanly destroyed)
+
+**Terminal state**: No Outbound Transitions
+
+
+### Transition State Definitions and Expected Behaviors
+
+Transition States exist to transition a node's [Managed Entities](#managed-entities) while allowing for node set up, teardown, and error recovery. As such, these states cannot be terminal states.
```

Comment:
This isn't as confusingly technocratic as the other time I made that remark, but I think this entire introduction section should be rewritten in more easily parsed language. I don't think so many words are needed to communicate what you want to communicate.

Reply:
As well for "Definition of a Transition"